src/Symfony/Bundle/WebProfilerBundle/Resources/views/Collector/logger.html.twig

```diff
@@ -55,28 +55,28 @@
         {% set filters = collector.filters %}
         <div class="log-filters">
             <div id="log-filter-type" class="log-filter">
-                <ul class="tab-navigation">
-                    <li class="{{ not has_error_logs and not has_deprecation_logs ? 'active' }}">
+                <div class="tab-navigation" role="tablist">
+                    <button role="tab" class="tab-control {{ not has_error_logs and not has_deprecation_logs ? 'active' }}">
                         <input {{ not has_error_logs and not has_deprecation_logs ? 'checked' }} type="radio" id="filter-log-type-all" name="filter-log-type" value="all">
                         <label for="filter-log-type-all">All messages</label>
-                    </li>
+                    </button>
 
-                    <li class="{{ has_error_logs ? 'active' }}">
+                    <button role="tab" class="tab-control {{ has_error_logs ? 'active' }}">
```

Comment:
This template puts an `input` inside a `button`, which is invalid HTML markup (and inaccessible one). You cannot have it being both a radio group and an ARIA tablist AFAICT.

Reply:
I'm going to fix it in a separate PR. Thanks.